worker/provision_device.json

```diff
@@ -1,29 +1,46 @@
 {
-    "name": "provision_device",
-    "description": "Provision device.",
-    "version": 2,
-    "tasks": [
-        {
-            "name": "create_device_inventory",
-            "type": "http",
-            "retries": 3,
-            "http": {
-                "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
-                "method": "POST",
-                "contentType": "application/json",
-                "body": "${workflow.input.device}",
-                "headers": {
-                    "X-MEN-RequestID": "${workflow.input.request_id}",
-                    "Authorization": "${workflow.input.authorization}"
-                },
-                "connectionTimeOut": 8000,
-                "readTimeOut": 8000
-            }
-        }
-    ],
-    "inputParameters": [
-        "request_id",
-        "authorization",
-        "device"
-    ]
+  "name": "provision_device",
+  "description": "Provision device.",
+  "version": 3,
+  "tasks": [
+    {
+      "name": "create_device_inventory",
+      "type": "http",
+      "retries": 3,
+      "http": {
+        "uri": "http://mender-inventory:8080/api/internal/v1/inventory/devices",
+        "method": "POST",
+        "contentType": "application/json",
+        "body": "{\"id\":\"${workflow.input.device_id}\"}",
+        "headers": {
+          "X-MEN-RequestID": "${workflow.input.request_id}",
+          "Authorization": "${workflow.input.authorization}"
+        },
+        "connectionTimeOut": 8000,
+        "readTimeOut": 8000
+      }
+    },
+    {
+      "name": "provision_deviceconnect",
+      "type": "http",
+      "retries": 3,
+      "http": {
+        "uri": "http://mender-deviceconnect:8080/api/internal/v1/deviceconnect/tenants/${workflow.input.tenant_id}/devices",
+        "method": "POST",
+        "contentType": "application/json",
+        "body": "{\"device_id\":\"${workflow.input.device_id}\"}",
```

Comment:
Looks good, but now that we introduced the "json" property for HTTP tasks, would you mind using it here instead of an interpolated body? Should be as simple as:

```
    ...
    "json": {
        "device_id": "${workflow.input.device_id}"
    },
    ...
```